LobbyServer2/LobbyServer/LobbyServerProtocol.cs

```diff
@@ -54,6 +54,7 @@ public BridgeServerProtocol CurrentServer
 
         public CharacterType OldCharacter { get; set; }
         
+        public int UsedGGBoostsCount { get; set; }
```

Comment:
There is already a variable for this in LobbyGameInfo

Reply:
Fixed